quickstart/cpp/windows/intent-recognition/README.md

```diff
@@ -4,7 +4,7 @@ This sample demonstrates how to recognize intent with C++ using the Speech SDK f
 
 ## Prerequisites
 
-* A LUIS account. You can get one for free through the [LUIS portal](https://www.luis.ai/home).
+* A LUIS account. [Create a new Azure account, and try Cognitive Services for free.](https://azure.microsoft.com/free/cognitive-services/)
```

Comment:
@v-jaswel  we should remove this change since it's referring to creating a LUIS account not a subscription. thanks!